views.py

```diff
@@ -67,46 +67,46 @@ async def atmpage(request: Request, lightning: str):
     # Decode and decrypt the 'p' parameter
```

Comment:
dead code
why "fixed" instead of being removed?

Reply:
atm page is not working in the current state, i leave this for reference what has to be done and tested, will change in later prs...